### PR TITLE
feature/50227_Update_account_information

### PR DESCRIPTION
**Pull request by luongdm-3080**
## Related Tickets
- [#50227](https://edu-redmine.sun-asterisk.vn/issues/50227)

## WHAT (optional)
- Edit information user.

## HOW
- I edit scss file.
- Add file edit registrations.

## Evidence (Screenshot or Video)
Screenshot from 2022-07-13 09-56-37
Screenshot from 2022-07-13 09-47-08

### CODE REVIEW

**Comment by luongdm-3080** [comment]
ready

File: app/views/devise/registrations/_modal_delete.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <div class="modal-content">
+      <div class="modal-header flex-column">
+        <div class="icon-box">
+          <i class="fa-solid fa-skull-crossbones"></i>
+        </div>
+        <h4 class="modal-title w-100"><%= t("sure_delete") %></h4>
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+      </div>
+      <div class="modal-body">
+        <p><%= t("danger_delete") %></p>
```
Comment (ghost) [inline app/views/devise/registrations/_modal_delete.html.erb]: bỏ ()

File: app/views/devise/registrations/_modal_delete.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<div id="myModal_delete<%= object.id %>" class="modal fade">
+  <div class="modal-dialog modal-confirm">
+    <div class="modal-content">
+      <div class="modal-header flex-column">
+        <div class="icon-box">
+          <i class="fa-solid fa-skull-crossbones"></i>
+        </div>
+        <h4 class="modal-title w-100"><%= t("sure_delete") %></h4>
```
Comment (ghost) [inline app/views/devise/registrations/_modal_delete.html.erb]: bỏ ()

File: app/views/devise/registrations/_modal_delete.html.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+      </div>
+      <div class="modal-body">
+        <p><%= t("danger_delete") %></p>
+      </div>
+      <div class="modal-footer justify-content-center">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal"><%= t("cancel") %></button>
```
Comment (ghost) [inline app/views/devise/registrations/_modal_delete.html.erb]: bỏ ()

File: app/views/devise/registrations/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,49 @@
+<button type="button" class="btn btn-delete" data-toggle="modal" data-target="#myModal_delete<%=@user.id%>">
+  <span class="mdi mdi-delete mdi-24px"></span>
+  <span class="mdi mdi-delete-empty mdi-24px"></span>
+  <i class="fa fa-trash"></i>
+  <span><%= t("delete-account") %></span>
```
Comment (ghost) [inline app/views/devise/registrations/edit.html.erb]: bỏ ()

**Review by ghost — COMMENTED** [review]
cmt

**Comment by luongdm-3080** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**